labs/firepower-restapi-111/1.md

```diff
@@ -1,42 +1,42 @@
 # FMC - REST - API - 111
 
-## Lab Objective
+## Objectives
 
-Basic understanding of Cisco Threat Intelligence Director (TID) APIs and their usage via API Explorer and Postman.  
+This lab teaches you the basic understanding of Cisco Threat Intelligence Director (TID) APIs and their usage via API Explorer and Postman.  
 
-Completion time: 30 minutes
 
 ## Prerequisites
-Basic understanding of REST principles
-Optional: Learning Lab "FMC - REST - API - 102" to get basic understanding of accessing FMC REST APIs using tokens.
+It's best if you already  have basic understanding of REST principles. Optionally, you can work through [Learning Lab "FMC - REST - API - 102](https://learninglabs.cisco.com/lab/firepower-restapi-102/step/1) before this lab, in order to get basic understanding of accessing FMC REST APIs using tokens.
+
+
+## Learning objective
+After completing this module you will have a basic understanding of the available Firepower Management Center (FMC) TID APIs and their usage.
 
-## Learning Objective
-After completing this module you will have a
-basic understanding of the available FMC TID APIs and their usage.
 
 ## FMC REST API Explorer
-The API Explorer provides the list of APIs available on the Firepower Management Center (FMC), their details and the ability to make permissible Rest API Calls using a simplified API console.
+The API Explorer provides the list of APIs available on the FMC, their details, and the ability to make permissible Rest API Calls using a simplified API console.
+
 
 ### Using API Explorer:
-1. Navigate to the URL: https://management_center_IP_or_name/api/api-explorer Or try the DevNet live sandbox Or try the DevNet live sandbox https://cisco-tid.cisco.com/api/api-explorer/
+1. Navigate to the https://management_center_IP_or_name/api/api-explorer URL. Or try the [DevNet Live Sandbox](https://cisco-tid.cisco.com/api/api-explorer/).
 
-2. Log in with your username and password and click "Sign In"
+2. Log in with your username and password and click "Sign In".
```

Comment:
We should probably say more about how people get the credentials for the DevNet Sandbox... as I understand it, they don't want to publish the actual credentials here though... let's find someone with a definitive answer.